code/modules/wod13/items/clothes/suit.dm

```diff
@@ -505,4 +505,383 @@
+	set category = "Object"
+	set src in usr
+	if(!istype(usr, /mob/living)||usr.stat)
+		return
+
+	if(tucked == 0)
+		tucked = 1
+		to_chat(usr, "<span class='notice'>You tuck in your your [src].</span>")
+	else
+		tucked = 0
+		to_chat(usr, "<span class='notice'>You untuck your [src].</span>")
+	update_icon_state()
+	usr.update_inv_wear_suit()
+
+/obj/item/clothing/suit/vampire/flannel/verb/button()
+	set name = "Toggle Shirt Buttons"
+	set category = "Object"
+	set src in usr
+	if(!istype(usr, /mob/living)||usr.stat)
+		return
+
+	if(buttoned == 0)
+		buttoned = 1
+		to_chat(usr, "<span class='notice'>You button your [src].</span>")
+	else
+		buttoned = 0
+		to_chat(usr, "<span class='notice'>You unbutton your [src].</span>")
+	update_icon_state()
+	usr.update_inv_wear_suit()
+
+
+/obj/item/clothing/suit/vampire/flannel/red
+	name = "red flannel shirt"
+	desc = "A comfy, red flannel shirt.  Unleash your inner hipster."
+	icon_state = "flannel_red"
+
+/obj/item/clothing/suit/vampire/flannel/aqua
+	name = "aqua flannel shirt"
+	desc = "A comfy, aqua flannel shirt.  Unleash your inner hipster."
+	icon_state = "flannel_aqua"
+
+/obj/item/clothing/suit/vampire/flannel/brown
+	name = "brown flannel shirt"
+	desc = "A comfy, brown flannel shirt.  Unleash your inner hipster."
+	icon_state = "flannel_brown"
+
+//Letterman jackets
+
+/obj/item/clothing/suit/vampire/letterman_c
+	name = "letterman jacket, UCLA"
+	desc = "A blue and gold UCLA varsity jacket."
+	icon_state = "letterman_c"
+
+/obj/item/clothing/suit/vampire/letterman_black
+	name = "black letterman jacket"
+	desc = "A letterman jacket in a moody black and white."
+	icon_state = "varsity"
+
+/obj/item/clothing/suit/vampire/letterman_purple
+	name = "purple letterman jacket"
+	desc = "A letterman jacket in a deep purple."
+	icon_state = "varsity_purple"
+
+//Military jacket
+
+/obj/item/clothing/suit/vampire/military_white
+	name = "white military jacket"
+	desc = "A white canvas jacket styled after classic American military garb. Feels sturdy, yet comfortable."
+	icon_state = "militaryjacket_white"
+
+/obj/item/clothing/suit/vampire/military_tan
+	name = "tan military jacket"
+	desc = "A tan canvas jacket styled after classic American military garb. Feels sturdy, yet comfortable."
+	icon_state = "militaryjacket_tan"
+
+/obj/item/clothing/suit/vampire/military_navy
+	name = "navy military jacket"
+	desc = "A navy canvas jacket styled after classic American military garb. Feels sturdy, yet comfortable."
+	icon_state = "militaryjacket_navy"
+
+/obj/item/clothing/suit/vampire/military_grey
+	name = "grey military jacket"
+	desc = "A grey canvas jacket styled after classic American military garb. Feels sturdy, yet comfortable."
+	icon_state = "militaryjacket_grey"
+
+/obj/item/clothing/suit/vampire/military_black
+	name = "black military jacket"
+	desc = "A black canvas jacket styled after classic American military garb. Feels sturdy, yet comfortable."
+	icon_state = "militaryjacket_black"
```

Comment:
1. these should be subtypes of ``/obj/item/clothing/suit/jacket/miljacket`` OR the existing military jacket should be a subtype of these.
2. these should all be a subtype of a generic military jacket, not all snowflaked